tests/publish_test.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from botocore.exceptions import ClientError
+
+from publish import upload_file
+
+
+class MockS3Client(object):
+    def upload_file(self, file_name, bucket, object_name):
+        if bucket == "private-bucket":
+            raise ClientError(
+                {'Error':
+                 {'Code': 'AccessDenied',
+                  'Message': 'Access Denied'}},
+                'PutObject',
+            )
+
+
+upload_file_cases = [
```

Comment:
This name is PEP8 compliance?

Reply:
At least, pycodestyle does not complain. What do you see wrong?

Reply:
Module variables ([constants](https://www.python.org/dev/peps/pep-0008/#constants)) must be in UPPER CASE.

Regarding pycodestyle, naming is [out of it's scope](https://pycodestyle.pycqa.org/en/latest/intro.html#disclaimer).

Reply:
As this variable isn't a constant, the name is ok.